server_cmds.go

```diff
@@ -351,19 +365,66 @@ func (s *ServerMethods) PrivilegeKeyAdd(ttype, id1, id2 int, options ...CmdArg)
 
 // OnlineClient represents a client online on a virtual server.
 type OnlineClient struct {
-	ID          int    `ms:"clid"`
-	ChannelID   int    `ms:"cid"`
-	DatabaseID  int    `ms:"client_database_id"`
-	Nickname    string `ms:"client_nickname"`
-	Type        int    `ms:"client_type"`
+	// Following variables are always returned by ClientList().
+	ID         int    `ms:"clid"`
+	ChannelID  int    `ms:"cid"`
+	DatabaseID int    `ms:"client_database_id"`
+	Nickname   string `ms:"client_nickname"`
+	Type       int    `ms:"client_type"`
+	// Following variables are optional and can be requested in ClientList() to get extended client information.
+	// note: Away and AwayMessage are currently optional but not using pointers for compatibility with older versions of teamspeakserver.
+	// This means they will always be returned, but their value will only be representative when the ClientAway option is specified.
+	OnlineClientAway   `ms:",squash"` // ts3.ClientAway
+	UniqueIdentifier   *string        `ms:"client_unique_identifier"` // ts3.ClientUid
+	OnlineClientVoice  `ms:",squash"` // ts3.ClientVoice
+	OnlineClientTimes  `ms:",squash"` // ts3.ClientTimes
+	OnlineClientGroups `ms:",squash"` // ts3.ClientGroups
+	OnlineClientInfo   `ms:",squash"` // ts3.ClientInfo
+	IconID             *int           `ms:"client_icon_id"`       // ts3.ClientIcon
+	Country            *string        `ms:"client_country"`       // ts3.ClientCountry
+	IP                 *string        `ms:"connection_client_ip"` // ts3.ClientIP
+	Badges             *string        `ms:"client_badges"`        // ts3.ClientBadges
```

Comment:
Similar updates to comments as above.

Reply:
Also here I followed your example and changed the comments.